contrib/scalar/scalar.c

```diff
@@ -913,7 +913,7 @@ static int add_or_remove_enlistment(int add)
 
 	return run_git(NULL, "config", "--global",
 		       add ? "--add" : "--unset",
-		       "--fixed-value", "scalar.repo",
+		       "scalar.repo",
```

Comment:
Really dropping `--fixed-value`? This option was created exactly for cases like this.

Reply:
```
12:16:01.574256 git.c:447               trace: built-in: git config --global --add --fixed-value scalar.repo C:/Users/me/repos/ForTests-123/src
error: --fixed-value only applies with 'value-pattern'
[...]
```

So yes, really dropping `--fixed-value` here. I keep it for the query, but not for setting it (because I want the command to succeed 😁).

Reply:
We need it for the `--unset` option. Perhaps this should be split with an `if (add)`.

Reply:
Oh, I see, we need it for the `--unset` case. But `--fixed-value` is incompatible with the `--add` case. So I replaced `"--fixed-value"` with `add ? "--no-fixed-value" : "--fixed-value"`.